runtimevar/paramstore/paramstore_test.go

```diff
@@ -12,11 +12,13 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+// Package paramstore 
```

Comment:
This can be removed as it's the same package paramstore

Reply:
Done.